modules/end-user-guide/examples/snip_che-IDE-default-yaml-file.adoc

```diff
@@ -0,0 +1,7 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+The default `devfile.yaml` file featuring settings for the supported IDEs is available link:https://github.com/eclipse-che/che-plugin-registry/blob/main/che-editors.yaml[here].
+====
```

Comment:
If you decide to include this snippet in the existing TIP:
```suggestion
* link:https://github.com/eclipse-che/che-plugin-registry/blob/main/che-editors.yaml[This `devfile.yaml` file] features the default devfile content for the supported IDEs.
====
```

Reply:
The suggested wording looks okay to me, in general. But I think we should add one important clarification here.

The [che-editors.yaml](https://github.com/eclipse-che/che-plugin-registry/blob/main/che-editors.yaml), in GitHub repo, is a kind of all-in-one yaml file containing all the editors Devfiles. So, actually, it's not a valid Devfile that can be passed to Che. This file serves as a source for generating the final Devfiles that are published to the public registry https://eclipse-che.github.io/che-plugin-registry/main/v3/plugins/

And the valid `devfile.yaml` files can be found in the corresponding folder of that public registry. E.g. https://eclipse-che.github.io/che-plugin-registry/main/v3/plugins/che-incubator/che-code/insiders

Reply:
sorry for not capturing this in the text, I'll rewrite it.

Reply:
@azatsarynnyy would something like this be ok? Or would you prefer me to mention the location of the valid devfiles instead?

[The `che-editors.yaml` file] features the devfiles of all supported IDEs.

Reply:
> [The che-editors.yaml file] features the devfiles of all supported IDEs.

@deerskindoll it sounds good to me